build/x86_64_sles12/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM opensuse:42.3
+FROM opensuse/leap:42.3
 
 RUN zypper -n refresh \
  && zypper -n install \
```

Comment:
While you're at it, do you mind bringing the rest of the commands below up-to-date with the CentOS 8 build Dockerfile (on [master](https://github.com/GoogleCloudPlatform/google-fluentd/blob/master/build/x86_64_centos8/Dockerfile))? Specifically:
- `git config` lines are not needed
- `procps` is not needed
- add `ruby`
- pull in the `gpg` key import command (with the comment)
- pull in the new `rvm` command line (with the comment)
Also, please
- move `zypper -n clean` to the very end, and
- add a comment in front of the two `sed` lines below, maybe along the lines of `# Pretend we're on SLES 12`...

Thanks.